src/main/java/fi/fmi/avi/model/Geometry.java

```diff
@@ -1,22 +1,25 @@
 package fi.fmi.avi.model;
 
+import java.math.BigInteger;
+import java.util.List;
+import java.util.Optional;
+
 import com.fasterxml.jackson.annotation.JsonSubTypes;
 import com.fasterxml.jackson.annotation.JsonTypeInfo;
 
 import fi.fmi.avi.model.immutable.CircleByCenterPointImpl;
 import fi.fmi.avi.model.immutable.PointGeometryImpl;
-import fi.fmi.avi.model.immutable.PolygonsGeometryImpl;
-
-@JsonTypeInfo(
-        use = JsonTypeInfo.Id.NAME,
-        include = JsonTypeInfo.As.PROPERTY,
-        property = "type"
-)
-@JsonSubTypes({
-        @JsonSubTypes.Type(value = PointGeometryImpl.class, name = "Point"),
-        @JsonSubTypes.Type(value = PolygonsGeometryImpl.class, name = "Polygon"),
-        @JsonSubTypes.Type(value = CircleByCenterPointImpl.class, name = "CircleByCenterPoint")
-})
+import fi.fmi.avi.model.immutable.PolygonGeometryImpl;
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, include = JsonTypeInfo.As.PROPERTY, property = "type")
+@JsonSubTypes({ @JsonSubTypes.Type(value = PointGeometryImpl.class, name = "Point"), @JsonSubTypes.Type(value = PolygonGeometryImpl.class, name = "Polygon"),
+        @JsonSubTypes.Type(value = CircleByCenterPointImpl.class, name = "CircleByCenterPoint") })
 
 public interface Geometry {
+
+    Optional<String> getSrsName();
+
+    Optional<BigInteger> getSrsDimension();
+
+    Optional<List<String>> getAxisLabels();
```

Comment:
I came to think if SRS fields should be extracted as distinct class `SpatialReferenceSystem` or `CoordinateReferenceSystem` or something like that? We could then provide constant instances for commonly used SRSes, like `WGS_84` / `EPSG_4326`, without need to (re)populate these fields whenever a Geometry instance is being built.

Reply:
This is now covered by issue #54.